public/index.html

```diff
@@ -24,6 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link
+      href="https://cdn.jsdelivr.net/gh/hung1001/font-awesome-pro@4cac1a6/css/all.css"
+      rel="stylesheet"
+      type="text/css"
+    />
     <link rel="stylesheet" href="https://robocon-ui.netlify.app/styles.css" />
 
     <title>React App</title>
```

Comment:
Cleanup the boilerplate comments, and also update the title of the page

Reply:
done!